Assets/Mirror/Runtime/Transport/Websocket/ClientJs.cs

```diff
@@ -40,9 +40,12 @@ public bool IsConnected
         int m_NativeRef = 0;
         readonly int id;
 
-        public Client()
+        static byte[] buffer;
+
+        public Client(int BufferSize)
         {
             id = Interlocked.Increment(ref idGenerator);
+            buffer = new byte[BufferSize];
```

Comment:
```suggestion
            buffer = buffer ?? new byte[BufferSize];
```

Do we need to reallocate this buffer every time we reconnect?  just use the previous buffer. 

You may want to check that the previous buffer is large enough though.

Although,  realistically,  why is the buffer static?  you only allocate it once when we connect,  nobody is connecting/disconnecting in a loop.

Reply:
The buffer is static because it has to be accessed by a MonoPInvokeCallback which must be static.

Reply:
How about this:
```
if (buffer == null || buffer.length < BufferSize)
{
    buffer = new byte[BufferSize];
}
```

because right now,  things might break if I do this:
```
var client1 = new Client(500);
var client2 = new Client(100);
```
Your buffer will be too small for client1

Reply:
another,  perhaps better approach is that you make the buffer non static,  and then get the buffer in the OnData callback for that client.